modules/apps/batch-engine/batch-engine-service/build.gradle

```diff
@@ -4,7 +4,7 @@ buildService {
 
 dependencies {
 	compileInclude group: "com.zaxxer", name: "SparseBitSet", version: "1.2"
-	compileInclude group: "org.apache.commons", name: "commons-collections4", version: "4.3"
+	compileInclude group: "org.apache.commons", name: "commons-collections4", version: "4.4"
```

Comment:
@jeffwu0724 Why you need to bump up the version from 4.3 to 4.4 for a pull that is supposed to move a lib to shared dependencies?

Reply:
When I add "commons-collections4" to shared-dependencies, I add it with version 4.4.
The reason I choose 4.4 here is that, in our code base, we are using 4.2, 4.3, and 4.4 at the same time. I decided to choose the highest version for easier maintenance in the future.
That's why I add "org.apache.commons:commons-collections4:4.4" in source-formatter, and auto sf at the end to make sure everyone is using version 4.4